README.md

```diff
@@ -24,13 +38,13 @@ optional arguments:
 
 The tool's behavior can be customized using a YAML configuration file. Take a look at `config.yaml` for an example. More documentation on this should be added in the future.
 
-
 **Results**:
-The tool will produce a results summary logged to stdout, and detailed test results along with its summary in json format.
+The tool will produce a results summary logged to stdout, and detailed test results along with its summary in json format in `outpu/output.json`.
```

Comment:
the output file is configured in the config file under the 'output' key, I wouldn't hardwire this, but explain where the file comes from.